content/en/docs/ops/configuration/extensibility/_index.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Extensibility
+description: Helps you manage WebAssembly based extensibility aspects of a running mesh.
+weight: 40
```

Comment:
This weight is the same as Observability. It should changed so that the ordering is correct. If you want it just before or after `Observability` you could make it 35 or 45.

Reply:
Updated. Thanks!